rbc fails to return in a function without arguments

```python
@omnisci('int64[]()')
def bar():
    a = Array(5, 'int64')
    for i in range(5):
        a[i] = 0
    return a
```

Numba fails to generate code for the function above:
```
E                   numba.core.errors.TypingError: Failed in nopython mode pipeline (step: nopython frontend)
E                   No conversion from (STRUCT__lPLbK)* to none for '$36return_value.1', defined at None
E
E                   File "rbc/tests/test_omnisci_array.py", line 427:
E                       def bar():
E                           <source elided>
E                               a[i] = 0
E                           return a
E                           ^
E
E                   [1] During: typing of assignment at /Users/guilhermeleobas/Documents/GitHub/rbc/rbc/tests/test_omnisci_array.py (427)
E
E                   File "rbc/tests/test_omnisci_array.py", line 427:
E                       def bar():
E                           <source elided>
E                               a[i] = 0
E                           return a
E                           ^
```

--- a/rbc/omniscidb.py
+++ b/rbc/omniscidb.py
@@ -364,6 +364,9 @@ class RemoteOmnisci(RemoteJIT):
             m = re.match(r'.*SQL Error: (.*)', msg.error_msg)
             if m:
                 raise OmnisciServerError(f'{m.group(1)}')
+            m = re.match(r'Could not bind *', msg.error_msg)
+            if m:
+                raise OmnisciServerError(msg.error_msg)
             # TODO: catch more known server failures here.
             raise
 

--- a/rbc/tests/test_omnisci_array.py
+++ b/rbc/tests/test_omnisci_array.py
@@ -1,6 +1,7 @@
 import os
 from collections import defaultdict
 from rbc.omnisci_backend import Array
+from rbc.errors import OmnisciServerError
 from numba import types as nb_types
 import pytest
 
@@ -462,5 +463,11 @@ def test_issue77(omnisci):
         a.fill(1)
         return a
 
-    _, result = omnisci.sql_execute('select issue77();')
-    assert list(result)[0][0] == [1, 1, 1, 1, 1]
+    if omnisci.version[:2] >= (5, 8):
+        _, result = omnisci.sql_execute('select issue77();')
+        assert list(result)[0][0] == [1, 1, 1, 1, 1]
+    else:
+        with pytest.raises(OmnisciServerError) as exc:
+            _, result = omnisci.sql_execute('select issue77();')
+
+        assert exc.match('Could not bind issue77()')

--- a/rbc/tests/test_omnisci_array.py
+++ b/rbc/tests/test_omnisci_array.py
@@ -452,3 +452,15 @@ def test_issue109(omnisci):
 
     _, result = omnisci.sql_execute('select issue109(3);')
     assert list(result) == [([0.0, 1.0, 2.0, 3.0, 4.0],)]
+
+
+def test_issue77(omnisci):
+
+    @omnisci('int64[]()')
+    def issue77():
+        a = Array(5, 'int64')
+        a.fill(1)
+        return a
+
+    _, result = omnisci.sql_execute('select issue77();')
+    assert list(result)[0][0] == [1, 1, 1, 1, 1]

--- a/rbc/tests/test_omnisci_udtf.py
+++ b/rbc/tests/test_omnisci_udtf.py
@@ -277,6 +277,7 @@ def test_raise_error(omnisci):
         omnisci.register()
 
     assert exc.match('raise statement is not supported')
+    omnisci.reset()
 
 
 def test_issue_235(omnisci):

--- a/rbc/tests/test_omnisci_udtf.py
+++ b/rbc/tests/test_omnisci_udtf.py
@@ -277,3 +277,24 @@ def test_raise_error(omnisci):
         omnisci.register()
 
     assert exc.match('raise statement is not supported')
+
+
+def test_issue_235(omnisci):
+
+    @omnisci('int32(Column<int64>, RowMultiplier, OutputColumn<int64>)')
+    def text_rbc_copy_rowmul(x, m, y):
+        for i in range(len(x)):
+            y[i] = x[i]
+        return len(x)
+
+    @omnisci('int32(Cursor<int64, double>, RowMultiplier, OutputColumn<int64>, OutputColumn<double>)')  # noqa: E501
+    def text_rbc_copy_rowmul(x, x2, m, y, y2):  # noqa: F811
+        for i in range(len(x)):
+            y[i] = x[i]
+            y2[i] = x2[i]
+        return len(x)
+
+    query = (f'select * from table(text_rbc_copy_rowmul('
+             f'cursor(select i8, f8 from {omnisci.table_name}), 1));')
+    _, result = omnisci.sql_execute(query)
+    assert list(result) == list(zip(np.arange(5), np.arange(5, dtype=np.float64)))

